software/mk/Makefile.verilog

```diff
@@ -57,6 +57,10 @@ $(info Machine executable: $(BSG_SIM_EXE))
 
 %.run: NBF_FILE=$*.nbf
 %.run: %.nbf
+# Rebuid the machine if machines dir is internal
+ifeq ($(abspath $(BSG_MACHINE_PATH)/..),$(abspath $(BSG_MANYCORE_DIR)/machines))
```

Comment:
$(abspath $(BSG_MACHINE_PATH)/..), isn't this referring to BSG_MANYCORE_DIR?

Reply:
https://github.com/bespoke-silicon-group/bsg_manycore/blob/b4e4cc7625e8318b20ae66006414a8b07703153e/software/mk/Makefile.paths#L6